src/richie/apps/courses/templates/courses/cms/fragment_organization_glimpse.html

```diff
@@ -2,6 +2,7 @@
 {% comment %}Obviously, the context template variable "organization" is required and must be an Organization page extension{% endcomment %}
 
 {% with organization_page=organization.extended_object organization_variant=organization_variant|default:"glimpse" %}
+<div>
```

Comment:
this change does not seem related to this PR?

Reply:
Probably not. Removing it.